File: app/src/main/java/com/moducode/gw2serveralarm/service/SharedPrefsManager.java
package com.moducode.gw2serveralarm.service;



/**
 * Created by Jay on 2017-11-09.
 */

public interface SharedPrefsManager {

    boolean isMonitoringServer();

    boolean isNotificationEnabled();

    void saveServerId(String id);

    String getSavedServerId();

    void saveServerName(String serverName);

    String getSavedServerName();

    void clearSavedTopic();


    boolean isVibrateEnabled();

    boolean isLedEnabled();

    String getAlarmUri();
}

File: app/src/main/java/com/moducode/gw2serveralarm/service/SharedPrefsManager.kt
package com.moducode.gw2serveralarm.service


/**
 * Created by Jay on 2017-11-09.
 */

interface SharedPrefsManager {

    fun isMonitoringServer(): Boolean

    fun isNotificationEnabled(): Boolean

    fun getSavedServerId(): String

    fun getSavedServerName(): String

    fun isVibrateEnabled(): Boolean

    fun isLedEnabled(): Boolean

    fun getAlarmUri(): String

    fun saveServerId(id: String)

    fun saveServerName(serverName: String)

    fun clearSavedTopic()
}
